addons/l10n_in/models/template_in.py

```diff
@@ -67,7 +67,7 @@ def _get_in_account_fiscal_position(self):
                 'sequence': 2,
                 'auto_apply': True,
                 'tax_ids': self._get_l10n_in_fiscal_tax_vals(),
-                'country_id': self.env.ref('base.in').id
+                'country_group_id': self.env.ref('l10n_in.inter_state_group').id,
```

Comment:
You don't need to call ref here already


```suggestion
                'country_group_id': 'l10n_in.inter_state_group',
```